setup.py

```diff
@@ -26,9 +26,9 @@
             'pytest-cov>=4,<5',
             'pytest-asyncio==0.*',
             'pytest-xdist==3.*',
-            'flake8==6.*',
-            'flake8-docstrings==1.*',
```

Comment:
You'll need to update Azure Pipelines, too

https://github.com/numat/ika/blob/13bf457f4776fe9ed2ea2cd7ea865541541c88b4/azure-pipelines.yml#L33-L35